tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -219,9 +219,29 @@ void TangoRosNode::onInit() {
   if (!node_handle_.hasParam(DATASET_UUID_PARAM_NAME)) {
     node_handle_.setParam(DATASET_UUID_PARAM_NAME, "");
   }
-  if (!node_handle_.hasParam(TANGO_3D_RECONSTRUCTION_RESOLUTION_PARAM_NAME)) {
-    node_handle_.setParam(TANGO_3D_RECONSTRUCTION_RESOLUTION_PARAM_NAME,
-                          TANGO_3D_RECONSTRUCTION_DEFAULT_RESOLUTION);
+  if (!node_handle_.hasParam(TANGO_3DR_RESOLUTION_PARAM_NAME)) {
```

Comment:
Could you add a helper function for this to avoid typos?
`set_default_value_if_parameter_does_not_exist()`